test/spec/SearchResults/getRefinements.js

```diff
@@ -107,11 +107,15 @@ test(
 
     var refinements = result.getRefinements();
 
-    t.deepEqual(refinements, [{
-      attributeName: 'hierarchicalCategories', count: 0, exhaustive: false,
-      name: 'Best Buy Gift Cards > Entertainment Gift Cards', type: 'hierarchical'
-    }]);
-
+    t.deepEqual(refinements, [
```

Comment:
Why the assertions change?

Reply:
because I fixed a bug in getRefinements with the reduce, as you can see in the JSON, there are supposed to be 17 hits, not 0